custom_components/idotmatrix/__init__.py

```
@@ -284,13 +295,119 @@ async def async_set_saved_design(call):
+
+    # Register show_bitcoin service
+    async def async_show_bitcoin(call):
+        """Handle the show_bitcoin service call."""
+        cfg = {
+            "price_entity": call.data.get("price_entity", "sensor.bitcoin_price"),
+            "change_entity": call.data.get("change_entity"),
+            "pixel_size": call.data.get("pixel_size"),
+        }
+        follow = call.data.get("follow", True)
+
+        for entry_id, coordinator in hass.data[DOMAIN].items():
```

Ruff (0.16.1)

[warning] Loop control variable `entry_id` not used within loop body

Rename unused `entry_id` to `_entry_id`

(B007)